src/outpack/location.py

```diff
@@ -71,6 +82,149 @@ def outpack_location_rename(old, new, root=None, *, locate=True):
     update_config(config, root.path)
 
 
+def location_resolve_valid(
+    location, root, include_local, include_orphan, allow_no_locations
+):
```

Comment:
I wonder about making these kw args with no defaults, that's how we're using them at least

```suggestion
def location_resolve_valid(
    location, root, *, include_local, include_orphan, allow_no_locations
):
```